tests/dofs/distribute_dofs_01.cc

```diff
@@ -0,0 +1,58 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2010 - 2021 by the deal.II authors
```

Comment:
It is sufficient to say 2021 here. :)

Reply:
True, but I think we don't have to be too strict about it here. :-)